### PR TITLE
StaticMethodCallExpr + TryCatchStatement

### PR DESCRIPTION
**Pull request by dafik**

### CODE REVIEW

File: src/CodeGen/Expr/StaticMethodCallExpr.php [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
+
+use CodeGen\ArgumentList;
 use CodeGen\Renderable;
-use CodeGen\Raw;
-use CodeGen\VariableDeflator;
-use LogicException;
 
-class MethodCall implements Renderable
+class StaticMethodCallExpr implements Renderable
 {
```
Comment (c9s) [inline src/CodeGen/Expr/StaticMethodCallExpr.php:8]: Maybe just `StaticMethodCall`?

Comment (dafik) [inline src/CodeGen/Expr/StaticMethodCallExpr.php:8]: Name is fixed  for file name and  autoloader compatibility. You can rename it for better fit project

**Comment by c9s** [comment]
I mean change the file name according to the class name. For MethodCall, FunctionCall... I have decided to remove the Expr suffix, which simplify the use.

**Comment by c9s** [comment]
FYI I've renamed the class in 960da42 :)

**Pull request merged**